gearman-defaults.sh

```diff
@@ -9,7 +9,7 @@
 
 DRUPAL_ROOT=${DRUPAL_ROOT:-/var/www/drupal7}
 DRUSH=${DRUSH:-`which drush`}
-CPU_COUNT=${CPU_COUNT:-`find /sys/devices/system/cpu -maxdepth 1 -type d -regex '.*/cpu[0-9]+$' | wc -l`}
+CPU_COUNT=${CPU_COUNT:-$(expr `find /sys/devices/system/cpu -maxdepth 1 -type d -regex '.*/cpu[0-9]+$' | wc -l` / 2)}
```

Comment:
Wondering if we should leave it count  -1 vs half? if a server has 8 CPU I would want more than 4 workers.

Reply:
/2 wasn't really part of this custom thing, I just did it cause it needs to be done, /2 is a much more conservative default than 1 worker per cpu and much less likely to cause issues, this is why I put it in a different PR.

The custom workers is also setup in such a way that it doesn't have to be installed unless it is required, so I don't want to make the default value for CPU_COUNT rely on it.

If more workers are desired it can be custom configured in /etc/default/gearman-workers this is just a conservative default. 

I was thinking just now there might be a decimal point but I guess expr auto rounds down the answer:

root@ak-vsphere-syd:/var/www/drupal7# expr 3 / 2
1